src/libraries/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -683,12 +683,12 @@ public static partial class Environment
         public static System.Version Version { get { throw null; } }
         public static long WorkingSet { get { throw null; } }
         [System.Diagnostics.CodeAnalysis.DoesNotReturnAttribute]
-        public static void Exit(int exitCode) { }
+        public static void Exit(int exitCode) => throw null; 
         public static string ExpandEnvironmentVariables(string name) { throw null; }
         [System.Diagnostics.CodeAnalysis.DoesNotReturnAttribute]
-        public static void FailFast(string? message) { }
+        public static void FailFast(string? message) => throw null;
```

Comment:
:memo: I'm not sure why there isn't a warning on `throw null`. I would expect `throw null!` would be required. Is this suppressed for the file or something?

Reply:
Yes, we suppress that for all reference assemblies. However, since this is autogenerated, this code will be removed by the autogenerated tool because the return type is `void`. 

We should fix that in our tool generation. I created: https://github.com/dotnet/arcade/issues/4820

Reply:
Yes:
```
eng\referenceAssemblies.props:    <!-- disable CS8597 because we throw null on reference assemblies. -->
eng\referenceAssemblies.props:    <NoWarn>$(NoWarn);CS8597</NoWarn>
```